src/content/docs/errors-inbox/errors-inbox.mdx

```diff
@@ -163,50 +165,10 @@ You can update the filter in errors inbox to show only errors assigned to yourse
-
-Once set up, you can jump from an error group directly to the offending code in your IDE by clicking the **Open in IDE** button.
-
-Learn more [here](docs/codestream/start-here/codestream-new-relic/#apm).
-
-## Connect an inbox to Jira [#jira]
-
-Connect errors inbox to Jira (Cloud) to easily create tickets for your errors, allowing for faster collaboration and resolution. Jira templates allow you to quickly create a ticket containing error details and links directly to the stack trace and APM for quick access and resolution. 
-
-We store a link to the the ticket alongside the associated error group for a period of time. If the error occurs again within that period, you can easily access associated tickets.
-
-<Callout variant="important">
-This integration supports Jira Cloud.
-</Callout>
-<Callout variant="important">
-This integration currently supports Atlassian-Jira Classic (company-managed) projects.
-</Callout>
-
-To connect an inbox to Jira:
-1. Click on the Jira integration icon on the far right side of the error group you want to connect to Jira. 
-  ![An image showing the location the jira integration button in errors inbox.](./images/jiraint.png "Jira integration button")
-  <figcaption>Clicking the Jira integration icon allows you to create a ticket based on a template, or create a template if you don't already have one.</figcaption>
-2. If you don’t already have a [connection to Jira](/docs/alerts-applied-intelligence/notifications/notification-integrations/#set-up-jira) set up in your account, click **Add JIRA Workspace** from the dropdown. Fill in all the fields and click **Test connection** before saving to ensure that your details are correct.
-3. Next, set up a template. Templates determine what information will be sent to Jira. Find more information about specific fields [here](/docs/alerts-applied-intelligence/notifications/notification-integrations/#configure-jira-message-template). Errors inbox does not currently support two-way communication with Jira, but you can select this option in case it is supported in the future.
-4. Once you have a template, click **Send test notification** to preview what the ticket looks like in Jira. If the preview looks good, click **Update message** to save the template.  Note that a test notification will create a Jira ticket in your Jira workspace.
-5. Now your team can create Jira tickets by clicking the Jira integration icon on the far right side of the error group and selecting a template.
+## Scoped views [#scoped]
 
+Scoped views allow you to view errors for entities in browser and mobile monitoring, without the need to create a workload. Access the errors inbox view for a specific entity by navigating to that entity in the browser or mobile monitoring view and selecting **Errors inbox** in the right nav. 
```

Comment:
@bradleycamacho, so users don't need to set up scoped views? They just need to navigate to them? This isn't too clear to me.